bluesky/bluesky-adaptive
# add: compose, readme, and dockerfile for qt gui

Adds the qtgui for the Queue Server from Bluesky widgets as an independent container.

## Description

Adds the image, docker compose, updated readme directions. Depends on [jozo/pyqt5](https://hub.docker.com/r/jozo/pyqt5). 

## Motivation and Context


People wanted better documentation around the gui, and having a quick container to spin up and test is the first step there. This should also facilitate better local testing. 

## Summary of Changes for Release Notes



Add container for Bluesky Widgets QTGui (queue-monitor) for Bluesky Adaptive


### Added
- New docker image and entry point

### Changed
- docker compose file
- README

## How Has This Been Tested?



Tested on a Mac which should be the hardest way to do this. 
- Follow readme. 
- Then docker compose up. 
- Use GUI to connect, then open environment. 
- Use GUI to add some agent driven naps to queue
- Start the queue


![Screenshot 2023-12-11 at 10 36 53 AM](https://github.com/bluesky/bluesky-adaptive/assets/43007690/33f90b91-6db1-48b0-9246-1f48dc09f721)
